### DISCUSSION TITLE
where is the generate router file

### DISCUSSION THREAD

**Discussion by alterhu2020** [nuxt/nuxt#16428] [Questions] [answered]
From the previous version nuxt2 we can view the generated routes based on the page directory in the file: `.nuxt/route.js`.  But currently in our new nuxt3, I cannot find this file. I want to verify the routes generated by nuxt is correct for me or not. 

Team, where I can get all the generated routes when runing nuxt3? thanks for help

**Comment by danielroe** [comment, marked as answer]
You can see all the generated files at `/_vfs`.

**Reply by alterhu2020** [reply]
> You can see all the generated files at `/_vfs`.

I didn't find this file in the project folder. 
<img width="426" alt="image" src="https://user-images.githubusercontent.com/5029046/200975697-0c3d234f-6e22-4303-89dd-404a33807fb8.png">

the `nuxt.config.ts` content as following:
```
import {defineNuxtConfig} from 'nuxt/config';

export default defineNuxtConfig({
    app: {
        head: {
            title: "Nuxt3 Template App",
            meta: [
                {name: "viewport", content: "width=device-width, initial-scale=1"},
                {
                    hid: "description",
                    name: "description",
                    content: "Nuxt3 Template App",
                },
            ],
            link: [{rel: "icon", type: "image/x-icon", href: "/favicon.ico"}],
        },
    },
    ssr: false,
    srcDir: "src/",
    css: ["@/assets/css/main.css"],
    modules: [
        ["@nuxtjs/tailwindcss"]
    ],
    sourcemap: true
});

```

**Reply by alterhu2020** [reply]
Please ignore it, i find view the router is from the generate url: `http://localhost:3000/_vfs/%23build%2Froutes`. thanks Team
